### PR TITLE
AppAuth 1.6 release

### PR DESCRIPTION
**Pull request by nonik0**
Includes support for service fabric environment and several bug fixes/improvements.

### CODE REVIEW

**Comment by nonik0** [comment]
@AlexGhiondea Ping. Please merge so we can release fixes/updates to customers.

File: sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj [hunk 1]
```diff
@@ -9,10 +9,8 @@
 		<![CDATA[
 			Documentation can be found at https://go.microsoft.com/fwlink/p/?linkid=862452.
 			
-			Support for new App Services managed identity API
-			Key vault client updates
-			Enabling CRL checks
-			Bug fixes
+			Support for managed identies on Service Fabric
+			Improvements and bug fixes for token providers
```
Comment (AlexGhiondea) [inline sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj:13]: Should the package version be incremented?

**Review by AlexGhiondea — APPROVED** [review]

File: sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj (hunk 1 above)
Comment (nonik0) [reply sdk/mgmtcommon/AppAuthentication/Azure.Services.AppAuthentication/Microsoft.Azure.Services.AppAuthentication.csproj:13]: It was incremented in an early PR to 1.6. That PR missed updating the AssemblyInfo.cs which is in this PR.

**Comment by nonik0** [comment]
@AlexGhiondea Ping. Could you merge the PR as well?

**Review by anaismiller — APPROVED** [review]

**Comment by jsquire** [comment]
Merging based on yesterday's request.

**Pull request merged**